noxfile.py

```diff
@@ -147,6 +147,34 @@ def test_integration_cloud(session):
     )
 
 
+@nox.session(
+    venv_backend=VENV_BACKEND,
+    python=environ.get("PYTHON_VERSION", "3.11"),
+)
+def test_integration_sqlachemy_v1(session):
+    """
+    Run integration tests on SQLAlchemy v1
+    (NOTE: the clickhouse-sqlalchemy driver only works with
+    SQLAlchemy 1.x)
+    """
+
+    # tests
+    _install(session, integration=True)
+    session.install(
+        "snowflake-sqlalchemy",
+        "redshift-connector",
```

Comment:
I haven't removed the snowflake or redhsift connectors because the sqlalchemy version is getting messed up. Installing version 1 explicitly is not helping either. I don't have an answer why, maybe we can handle it in a separate PR.

Reply:
weird, ok. please open a separate PR.